fuse.py

```diff
@@ -26,6 +26,8 @@
 
 import logging
 
+logging.basicConfig(level=logging.INFO)
```

Comment:
I'm not really sure this should be done at the library level. Generally I would leave this up to the consumer.